### ISSUE TITLE
Hebrew assets are missing but audio table is present

### ISSUE THREAD

**Issue by ThM112** [GaijinEntertainment/fmod_studio_warthunder_for_modders#115]
basically there are no Hebrew voice lines in the current version of the sound files, but the audio table for the language he (Hebrew) is present and empty

**Comment by Uchuujin01** [comment]
Yes, it's like this for now. I'm about to add all crew voices, but I don't know when, I should spare for this some significant time.
